metabase/metabase
# Extract Cypress user data

### Status
READY

### What does this PR accomplish?
- Extracts all user-related data into a separate file
    - That data is needed both for `commands.js` and also for individual tests
    - Since `cypress.js` imports `commands.js`, this data had to be extracted into a separate file
- Converts user-related functions into a set of Cypress custom commands
- Updates all tests with the related changes

### How to verify this works?
- All tests should pass in CI
- Additionally, we need to verify that smoke tests still work:
    - `yarn test-cypress-no-build --folder frontend/test/metabase-smoketest/`

### Additional info:
- I am open for suggestions on what would be the right name for the new file
- I used `cypress_data` because it could possibly contain other shared data, not just user-related stuff (for example, `SAMPLE_DATASET_ID`)
